### ISSUE TITLE
JSON Viewer crashes

### ISSUE THREAD

**Issue by NN---** [sunjw/jstoolnpp#34]
Version 1.20

Write simple JSON:

```
{}
```

Select JSON viewer and it crashes.

**Comment by sunjw** [comment]
I cannot reproduce this. 
![nocrash](https://cloud.githubusercontent.com/assets/846335/21080467/e83c863a-bfea-11e6-95cc-19923ee4daba.png)

Could you please paste your screenshot?

**Comment by NN---** [comment]
I use x64 build

**Comment by sunjw** [comment]
Still no crash found on x64 build @NN--- 
![nocrash x64](https://cloud.githubusercontent.com/assets/846335/21080631/ba3fd2ba-bfee-11e6-9fd8-2c3d6e9834e0.png)

Does it crash on other JSON file?

**Comment by NN---** [comment]
Yes

**Comment by NN---** [comment]
UINT wParam, LONG lParam ? :)

Use WPARAM and LPARAM types which are defined correctly for you: https://msdn.microsoft.com/en-us/library/windows/desktop/ms633573(v=vs.85).aspx

Crashes here
```cpp
LRESULT CALLBACK EditControlEx::EditControlProc(
	HWND hEditControl, UINT message, UINT wParam, LONG lParam)
{
	EditControlEx *pEditControlEx = (EditControlEx *)(::GetWindowLongPtr(hEditControl, GWLP_USERDATA));

	switch (message)
    {
	case WM_KEYDOWN:
		{
			if((::GetKeyState(VK_CONTROL) & 0xFF00) == 0xFF00)
			{
				if(wParam == 'A' || wParam == 'a')  
				{  
					::SendMessage(pEditControlEx->m_hEditControl, EM_SETSEL, 0, -1);
					return TRUE;  
				}  
			}
		}
		break;
	}

	return CallWindowProc( // <<=== HERE
		(WNDPROC)pEditControlEx->m_oldEditControlProc, 
		hEditControl, message, wParam, lParam);
}
```

**Comment by sunjw** [comment]
OK, thanks for the fix. I just merge it.

**Comment by sunjw** [comment]
Fixed with dae70b890bd774e57d3dd0a73207a42a534c19dd

**Comment by sunjw** [comment]
Hi, @NN--- 
Would you like to try this build and see if any crash happens?

![JSMinNPP.dll](https://cloud.githubusercontent.com/assets/846335/21080964/3b1bbcb6-bff7-11e6-9a8b-239881af636d.png)

(Rename it to "JSMinNPP.dll")

**Comment by NN---** [comment]
Works

**Comment by sunjw** [comment]
Thanks very much.

**Comment by sunjw** [comment]
Fixed in and after version 1.20.2. 
Download from:
http://www.sunjw.us/jstoolnpp/download.php
